src/routes/embed/documents/[id]/pages/[page]/+page.svelte

```diff
@@ -169,7 +169,7 @@
     or
     <a
       style="color: #5a76a0; text-decoration: underline;"
-      href={textUrl(doc, page - 1).toString()}
+      href={textUrl(doc, page).href}
```

Comment:
Nothing wrong here, just flagging how the inconsistency in whether a page number is zero-indexed or not can produce bugs.

Reply:
Yes. It's super annoying. I think it's been an issue since the beginning.